sdk/Pulumi/Pulumi.csproj

```diff
@@ -31,10 +31,10 @@
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.24.0" />
     <PackageReference Include="Ben.Demystifier" Version="0.4.1" />
-    <PackageReference Include="Grpc.Net.Client" Version="2.51.0" />
-    <PackageReference Include="Grpc.AspNetCore.Server" Version="2.51.0" />
-    <PackageReference Include="Grpc.AspNetCore.Server.Reflection" Version="2.51.0" />
-    <PackageReference Include="Grpc.Tools" Version="2.51.0">
+    <PackageReference Include="Grpc.Net.Client" Version="2.52.0" />
```

Comment:
I was curious why this was upgraded while all the other libraries are getting downgraded.  But it looks like this got downgraded for some reason when upgrading gRPC, so it looks like the right thing to do.